lib/form/html.js

```diff
@@ -29,7 +29,46 @@ const addError = (html, input, errors) => {
     });
   }
 
-  result += html;
+  result += render;
+
+  return result;
+};
+
+/*
+ * Add Required Rendered Inputs
+ *
+ * @param {string} html - Rendered HTML
+ * @param {InputType} type - Input type being rendered
+ *
+ * @returns {string} - Rendered HTML, with required indicators
+ */
+const addRequired = (html, input) => {
+  let render = html;
+  let result = '';
+
+  Object.keys(input.inputs).map(inp => {
+    if (((input.hasOwnProperty('required')) && input.required === 'save') || ((input.inputs[inp].hasOwnProperty('required')) && input.inputs[inp].required === 'save')) {
+      const id = input.inputs[inp].id;
+      let level = 'required ';
+      if (input.inputs[inp].hasOwnProperty('required')) {
+        level += `required--${input.inputs[inp].required}`;
+      }
+      else {
+        level += `required--${input.required}`;
+      }
+
+      // add required class to label
+      let find = new RegExp(`for=['"]\\s*${id}\\s*['"]`);
```

Comment:
You're going to have to do better than this. If the label has a class attribute already, whichever one comes [first will win](http://codepen.io/Snugug/pen/GqojpN). You need to check to find the `for` _and_ see if it has `class` already. If it does not, then you can add this. Otherwise, you need to attach it to the existing `class` attribute